mne/inverse_sparse/mxne_inverse.py

```diff
@@ -282,7 +282,7 @@ def mixed_norm(evoked, forward, noise_cov, alpha, loose='auto', depth=0.8,
                maxit=3000, tol=1e-4, active_set_size=10, pca=True,
                debias=True, time_pca=True, weights=None, weights_min=None,
                solver='auto', n_mxne_iter=1, return_residual=False,
-               return_as_dipoles=False, verbose=None):
+               return_as_dipoles=False, dgap_freq=10, verbose=None):
```

Comment:
dgap_freq needs to be passed to the solver functions for MxNE

Reply:
Setting the default to 10 here changes the current behavior of the MxNE solvers, which compute the gap in every iteration currently.

Reply:
We have to check that dgap_freq is > 0. dgap_freq = 0 will otherwise result in a division by 0